### PR TITLE
update disk section, add SMART and drive power down howtos. Fixes #164 and #159

### LINKED ISSUE #159: document the disk power status columns
Thanks to forum user @phil47 for highlighting this issue. We have no documentation on the disk idle power down feature recently added to the disk page. This could be added as a section to the disks page:
http://rockstor.com/docs/disks.html
ie an explanation of the pause and hour glass icon functions and how these might relate to the APM level in the next column. It would also be good to explain that the hdparm method used to configure this behaviour is not guaranteed to work with all hardware.

Please update the following forum thread with progress on this issue:
https://forum.rockstor.com/t/veille-des-disque-dur/2373

### LINKED ISSUE #164: Update disk section
The Disks page is now outdated given changes in disk: naming, power status and custom SMART options. There have also been recent changes to enable disk 'Roles' which in part enables some partition capability, as well as changing some basic functions such as disk and more recently partition wiping. This issue is intended to address the changes needed to bring the disk page back up to date.

This issue is also intended as a super set of:
"document the disk power status columns" #159

### PR DESCRIPTION
**Pull request by phillxnet**
Update the Disks page to cover the following changes in:
https://github.com/rockstor/rockstor-core

- "revise internal use and format of device names. Fixes #1320 #1357" (https://github.com/rockstor/rockstor-core/pull/1357)
- partial coverage of "enhance disk role/management subsystem. Fixes #1494 #1622" (https://github.com/rockstor/rockstor-core/pull/1622)
- "drive power down interface. Fixes #885 #1292" (https://github.com/rockstor/rockstor-core/pull/1292)
- "add custom smart parameter option. Fixes #1079 #1221" (https://github.com/rockstor/rockstor-core/pull/1221)
- various SMART display and parsing improvements

Note that elements of this pr pertain to testing channel updates only (roles config primarily) as of this pr's submission date.

Tested using 64bit Sphinx v1.4.8.
No errors indicated.

The resulting html (via "make html") was tested in Chrome and Firefox and rendered as intended.

In summary this pr:
- updates the whole disk page and links to:
- added SMART HowTo
- added Disk Power Down HowTo

Fixes #164 
Fixes #159 

Additional internal links to the new material could be made but are reserved for future pr's.

@schakrava Ready for review.

### CODE REVIEW

**Comment by phillxnet** [comment]
@schakrava Now that 3.9.0 stable is out and we are into the next testing channel updates I think it would be a good time for this to be merged and published. Covers most of the recent disk role changes and some long overdue additions.

**Comment by schakrava** [comment]
Thank you @phillxnet . As expected, really quality docs contribution!

**Comment by phillxnet** [comment]
@schakrava Cheers and thanks.
